audit/sst-14.0.audit

```diff
@@ -0,0 +1,55 @@
+SST-Core Version (14.0.0)
+Test project /Users/kgriesser/work/sst-ext-tests/build
```

Comment:
@kpgriesser wasn't really sure what to do with the *.audit files - they contain path information that won't generalize across different machines. suspect this is overcome by events (OBE) at this point.